src/shared_utilities/Domains.jl

```diff
@@ -721,6 +746,27 @@ function get_Δz(z::ClimaCore.Fields.Field)
     return Δz_top ./ 2, Δz_bottom ./ 2
 end
 
+"""
+    get_z_sfc(z::ClimaCore.Fields.Field)
+
+A function to return the coordinates of the surface `z`, projected onto the one-/two-dimensional surface space.
+This space is distinct from the level space of `z`.
+"""
+function get_z_sfc(z::ClimaCore.Fields.Field)
```

Comment:
Could we rename this to a more general `top_face_to_surface`, analogous to `top_center_to_surface`? It seems like the only part specific to `z` here is `z_face = ClimaCore.Fields.coordinate_field(fs).z`, so maybe we could keep this function and just have it pass that `z_face` to the generalized `top_face_to_surface` function?

Reply:
This is a good idea! But I realized there is an easier way to get z_sfc, and since we dont need the `top_face_to_surface` function otherwise, I didnt add it for now. currently we just do:
    `z_sfc = ClimaCore.Fields.coordinate_field(surface_space).z`

Reply:
edit- that didnt work. so I added in top_face_to_surface